pyswmm/toolkitapi.py

```diff
@@ -531,3 +531,52 @@ class SMO_systemAttribute(Enum):
     outfall_flows = 11
     volume_stored = 12
     evap_rate = 13
+
+
+subcatch_attribute = dict(
```

Comment:
I see you are creating a dictionary to hold attributes. What value is this adding when attributes are already in enum classes?